crates/bevy_internal/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "bevy_internal"
 version = "0.9.0"
 edition = "2021"
-description = "An internal Bevy crate used to facilitate optional dynamic linking via the 'dynamic' feature"
+description = "An internal Bevy crate used to facilitate optional dynamic linking via the 'dynamic_linking' feature"
```

Comment:
```
`dynamic_linking`
```
?

Reply:
I'm not sure if cargo supports markdown for crate desc, could you give me an example?